src/js/videojs-dash.js

```diff
@@ -1,47 +1,45 @@
-'use strict';
-var window_ = require('global/window');
-var videojs = require('video.js');
-var dashjs = require('dashjs');
-
-  var
-    isArray = function(a) {
-      return Object.prototype.toString.call(a) === '[object Array]';
-    },
-    isObject = function (a) {
-      return Object.prototype.toString.call(a) === '[object Object]';
-    },
-    mergeOptions = function(obj1, obj2){
-      var key, val1, val2, res;
-
-      // make a copy of obj1 so we're not overwriting original values.
-      // like prototype.options_ and all sub options objects
-      res = {};
-
-      for (key in obj2){
-        if (obj2.hasOwnProperty(key)) {
-          val1 = obj1[key];
-          val2 = obj2[key];
-
-          // Check if both properties are pure objects and do a deep merge if so
-          if (isObject(val1) && isObject(val2)) {
-            obj1[key] = mergeOptions(val1, val2);
-          } else {
-            obj1[key] = obj2[key];
-          }
+import window from 'global/window';
+import videojs from 'video.js';
+import dashjs from 'dashjs';
```

Comment:
The browserify shim should map dashjs imports to window.dashjs so in the end you would include the dash.js, video.js and videojs-contrib-dash scripts in your page or do this in the script you want to browserify:
```javascript
var videojs = require('video.js');
window.videojs = videojs;
window.dashjs = require('dashjs');
require('videojs-contrib-dash');

var player = videojs('player');

player.ready(function() {
  player.src({
    src: 'http://dash.edgesuite.net/envivio/EnvivioDash3/manifest.mpd',
    type: 'application/dash+xml'
  });
});
```

I'm going to see if I can have the browserify-shim only apply when building the `dist/` files and not when you import the plugin so you don't have to do the weird `window` workaround.